sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -460,6 +462,37 @@ def run_tests_without_pytest():
     runner.run(suite)
 
 
+def test_check_class_weight_balanced_linear_classifier():
+    class BadBalancedWeightsClassifier(BaseBadClassifier):
```

Comment:
You can move this class outside next to the other classes.

Reply:
Hi @glemaitre, thank you for reviewing this.
I changed the location of the class as you suggested. Can you point me towards where can I find "what's new" to add an entry ?

Reply:
In this file:
https://github.com/scikit-learn/scikit-learn/blob/master/doc/whats_new/v0.21.rst